glocal_exploration/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(glocal_exploration)
 
-set(CMAKE_CXX_STANDARD 17)
```

Comment:
Some IDEs (such as clion) can for some reason not parse add_definitions(-std=c++17) but can parse set(CMAKE_CXX_STANDARD 17). It's only for certain IDEs though and should do totally the same thing when building. Maybe for the sake of maintainability might be easier to keep the set(CM...)? Although it would potentially be prettier otherwise.

Reply:
Nvm, got things sorted out and either works. Feel free to merge.

Reply:
Great!